### PR TITLE
Some text renderer javadoc

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

File: mappings/net/minecraft/client/font/TextRenderer.mapping [hunk 1]
```diff
@@ -65,9 +75,15 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
 		ARG 9 underlineColor
 		ARG 10 light
 	METHOD method_1726 isRightToLeft ()Z
+		COMMENT Checks if the current language the text renderer is can render uses right to left writing.
```
Comment (YanisBft) [inline mappings/net/minecraft/client/font/TextRenderer.mapping]: I don't understand the phrasing here

File: mappings/net/minecraft/client/font/TextRenderer.mapping [hunk 2]
```diff
@@ -191,6 +216,7 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
 			COMMENT the text color in the 0xAARRGGBB format
 		ARG 6 rightToLeft
 	METHOD method_30880 getWidth (Lnet/minecraft/class_5481;)I
+		COMMENT Gets the width of some text when rendered
```
Comment (YanisBft) [inline mappings/net/minecraft/client/font/TextRenderer.mapping]:
```suggestion
		COMMENT Gets the width of some text when rendered.
```

File: mappings/net/minecraft/client/font/TextRenderer.mapping [hunk 3]
```diff
@@ -27,10 +29,18 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
...
-	METHOD method_1713 getStringBoundedHeight (Ljava/lang/String;I)I
+	METHOD method_1713 getWrappedLinesHeight (Ljava/lang/String;I)I
+		COMMENT Gets the height of the text when it has been wrapped.
+		COMMENT
+		COMMENT @return the height of the wrapped text
+		COMMENT @see TextRenderer#wrapLines(StringVisitable, int)
 		ARG 1 text
 		ARG 2 maxWidth
 	METHOD method_1714 trimToWidth (Lnet/minecraft/class_5348;I)Lnet/minecraft/class_5348;
+		COMMENT Trim a string to be at most {@code maxWidth} wide.
```
Comment (enbrain) [inline mappings/net/minecraft/client/font/TextRenderer.mapping]:
```suggestion
		COMMENT Trims a string to be at most {@code maxWidth} wide.
```

File: mappings/net/minecraft/client/font/TextRenderer.mapping [hunk 4]
```diff
@@ -141,14 +157,23 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
 		ARG 10 light
 		ARG 11 rightToLeft
 	METHOD method_27523 trimToWidth (Ljava/lang/String;I)Ljava/lang/String;
+		COMMENT Trim a string to be at most {@code maxWidth} wide.
```
Comment (enbrain) [inline mappings/net/minecraft/client/font/TextRenderer.mapping]:
```suggestion
		COMMENT Trims a string to be at most {@code maxWidth} wide.
```

File: mappings/net/minecraft/client/font/TextRenderer.mapping [hunk 5]
```diff
@@ -141,14 +157,23 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
...
+		COMMENT
+		COMMENT @return the trimmed string
+		COMMENT @see TextHandler#trimToWidth(String, int, Style)
 		ARG 1 text
 		ARG 2 maxWidth
 	METHOD method_27524 trimToWidth (Ljava/lang/String;IZ)Ljava/lang/String;
+		COMMENT Trim a string to be at most {@code maxWidth} wide.
```
Comment (enbrain) [inline mappings/net/minecraft/client/font/TextRenderer.mapping]:
```suggestion
		COMMENT Trims a string to be at most {@code maxWidth} wide.
```

File: mappings/net/minecraft/client/font/TextRenderer.mapping (hunk 1 above)
Comment (mounderfod) [reply mappings/net/minecraft/client/font/TextRenderer.mapping]: `Checks if the language being used by the renderer uses right to left` might be better?

**Review by YanisBft — APPROVED** [review]

**Review by mounderfod — APPROVED** [review]
Looks good, javadoc is well written. 👍

**Review by enbrain — APPROVED** [review]

**Pull request merged**